dist/extras/named-register.js

```diff
@@ -40,4 +40,4 @@
   systemJSPrototype.instantiate = function (url, firstParentUrl) {
     return this.registerRegistry[url] || instantiate.call(this, url, firstParentUrl);
   };
-})();
\ No newline at end of file
+})();
```

Comment:
The dist directory is in version control, but you shouldn't push changes to it in PRs -- Guy Bedford runs the build right before publishing and that's the only time the dist directory files should change.

Reply:
That makes Waaaay more sense. I'll update that tonight. I didn't add (or try to add) any tests yet, so I'd also like to look into that